erpnext/hr/doctype/expense_claim_detail/expense_claim_detail.json

```diff
@@ -170,41 +233,80 @@
    "width": "300px"
   }, 
   {
+   "allow_bulk_edit": 0, 
+   "allow_on_submit": 0, 
+   "bold": 0, 
+   "collapsible": 0, 
+   "columns": 0, 
+   "fieldname": "attachment", 
```

Comment:
The reason I have it here is because this links each line item with an image. If I'm trying to verify the information in each line item of the expense report, being able to quickly see which attachment (supporting evidence for the claim) is associated with the claim is crucial. If there are 10 line items and 10 attachments without any way of knowing which is related to which, it becomes much more cumbersome.